dockerfiles/steps/git-fetch.bash

```diff
@@ -1,6 +1,4 @@
 parse_book_dir
-[[ "$ARG_GIT_REF" == latest ]] && ARG_GIT_REF=main
-[[ "$ARG_REPO_NAME" == */* ]] || ARG_REPO_NAME="openstax/$ARG_REPO_NAME"
```

Comment:
Does something else prefix `openstax/` to the repo argument? For some reason I thought devs and CMs sometimes omitted the `openstax/` when starting a job

Reply:
I will double check, but I think this is the place where `openstax/` was automatically being added if it was omitted.

Reply:
This appears to be the only place that prefixed `openstax/` when the repo owner is not included. I moved it into docker-entrypoint so that the `ARG_REPO_NAME` variable will always include the repo owner. Right now the variable is only used in `git-fetch`.